ios/sound-send/sound-send/ViewController.swift

```diff
@@ -9,17 +9,98 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    //the base url, change it when we have one i guess?
+    let baseurl = "http://www.soundsend.com"
```

Comment:
Yeah mine is currently in the same format for the base url. We can try Aditi's plan of declaring it as a class function.